### DISCUSSION TITLE
07_PyTorch_Experiment_Tracking Access TensorBoard data as DataFrames

### DISCUSSION THREAD

**Discussion by ClivanL** [mrdbourke/pytorch-deep-learning#805] [Q&A]
Hi, I am trying to load my tensorboard data into dataframe using the guide provided in the url: 
https://www.tensorflow.org/tensorboard/dataframe_api

From my understanding, I am supposed to upload my data into TensorBoard.dev and then retrieve the data using the code below using the experiment id:

```
experiment_id = "c1KCv3X3QvGwaXfgX1c4tg"
experiment = tb.data.experimental.ExperimentFromDev(experiment_id)
df = experiment.get_scalars()
df
```

However, i went to the site and it seems like the site has been shut down as of 1 Jan 2024. Is there anyway I can achieve the following without uploading my data to the site?

**Comment by jh-jensen** [comment]
November 11th here, still no answer? Having the same issue although it's not the most important thing in the world.

**Comment by LuluW8071** [comment]
You can try other experiment trackers like CometML, Weight and Bias, MLFlow. The interface is much better and and you can set online tracker as well as offline or both.
